src/plugins/jartic/__init__.py

```
@@ -0,0 +1,6 @@
+from ..import PluginRegistry
+from .datasource import JARTICDataSource
+
+
+def register_plugin(registry: PluginRegistry) -> None:
+    registry.register('jartic', JARTICDataSource)
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)